guides/miner/hiveos.mdx

```diff
@@ -174,11 +172,11 @@ Now that all of our dependencies are installed, we can install quai-gpu-miner. T
 
 ### Run
 
-To run the miner, you'll need a quai-stratum-proxy to connect to. Visit the [quai-stratum-proxy](/guides/client/stratum) docs for information on how to install and configure it. The proxy configuration will determine which shard your gpu-miner is running on and the address payouts are awarded to.
+To run the miner, you'll need a Stratum proxy to connect to. Visit the [quai-stratum-proxy](/guides/client/stratum) docs for information on how to install and configure it. The proxy configuration will determine which shard your gpu-miner is running on and the address payouts are awarded to.
 
-First, you'll need to obtain the IP Address and port your proxy is running on. The default port is `3333`.
+First, you'll need to obtain the IP Address and port your proxy is running on from the Stratum proxy logs. The default port is `3333`.
 
-Once you have the address and port and are in the `build` directory, run the following command to start the miner:
+Once you've obtained the IP Address and port, run the miner with the following command, making sure to replace `PROXYIPADDRESS` with the IP Address and `STRATUMPORT` with the port your proxy is running on:
```

Comment:
Can do this in a future PR, but don't forget the note about localhost being the relevant IP for machines running both stratum and the miner. If not this PR then please make an issue